packages/email-auth/__tests__/generate-otp.spec.ts

```diff
@@ -17,32 +18,31 @@ describe('POST /otp/generate', () => {
           type: String,
           index: 1,
           required: true,
-        }
+        },
       }),
     );
 
     await userModel.create({
-      emailAddress
+      emailAddress,
     });
 
     const emailAdapter = {
-      sendEmailOtp: jest.fn(async () => {})
+      sendEmailOtp: jest.fn(async () => {}),
     };
-    
+
     const server = new EmailAuthServer(
-      ctx.server,
       new MongooseStorageAdapter(ctx.mongoose, {
-        userModel: 'User'
+        userModel: 'User',
       }),
-      emailAdapter
+      emailAdapter,
     );
 
-    await server.init();
+    ctx.server.on('request', server.expressMiddleware());
```

Comment:
install `express` as dev dependency and use that on this test

Reply:
fixed